src/github.com/matrix-org/dendrite/clientapi/writers/sendmessage.go

```diff
@@ -14,7 +15,9 @@ type SendMessage struct {
 // OnIncomingRequest implements util.JSONRequestHandler
 func (s *SendMessage) OnIncomingRequest(req *http.Request) (interface{}, *util.HTTPError) {
 	logger := req.Context().Value(util.CtxValueLogger).(*log.Entry)
-	logger.Info("Doing stuff...")
+	vars := mux.Vars(req)
+	roomID := vars["roomID"]
+	logger.WithField("roomID", roomID).Info("Doing stuff...")
```

Comment:
It is "interesting" and I'm wondering how best to handle that going forwards. In all practicalities we WILL be using a gorilla mux so I don't feel inclined to doc this every time we pluck something out of vars.

I dislike how separated the path pattern is from where we access things, but I like how we can easily see the endpoints available in routing.go